# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fgetty/hotfix-makefile.patch
# Copyright (C) 2004 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- fgetty-0.7/Makefile.vanilla	2024-01-12 19:41:02.599000000 +0100
+++ fgetty-0.7/Makefile	2024-01-12 19:41:22.779000000 +0100
@@ -8,12 +8,13 @@
 #CROSS=arm-linux-
 CROSS=
 LDFLAGS=-s
+STRIP=strip
 
 %.o: %.c
 #	gcc -march=i386 -mcpu=i386 -pipe -Os -fomit-frame-pointer -I../dietlibc/include -c $^ -DTEST
 	$(DIET) $(CROSS)$(CC) -pipe -Os -fomit-frame-pointer -I../dietlibc/include -c $^ -DTEST
 #	gcc -march=i386 -mcpu=i386 -pipe -g -I../dietlibc/include -DTEST -c $^
-	$(CROSS)strip -x -R .comment -R .note $@
+	$(CROSS)$(STRIP) -x -R .comment -R .note $@
 
 %: %.o
 	$(DIET) $(CROSS)$(CC) -o $@ $^ $(LDFLAGS)
